app/shared/connectors/httpparsers/HttpParser.scala

```diff
@@ -64,13 +64,20 @@ trait HttpParser extends Logging {
   private val multipleErrorCodesInResponseReads: Reads[Seq[DownstreamErrorCode]] =
     (__ \ "response").read[Seq[JsObject]].map(_.map(obj => DownstreamErrorCode((obj \ "errorCode").as[String])))
 
+  private val multipleFailureErrorTypesReads: Reads[Seq[DownstreamErrorCode]] =
```

Comment:
Please remove all references to this as this type of reads is unmappable for this endpoint. The multipleTopLevelErrorCodesReads and multipleErrorCodesInResponseReads are the ones that are applicable to this endpoint. Thanks